assets/js/base/components/formatted-monetary-amount/index.tsx

```diff
@@ -64,7 +74,7 @@ const FormattedMonetaryAmount = ( {
 		className
 	);
 	const numberFormatProps = {
-		displayType,
+		displayType: 'text' as NumberFormatProps[ 'displayType' ],
```

Comment:
I think we should move this one to our `FormattedMonetaryAmount` props since we do use this in other places and we always set the default value to 'text'. The type can then go within `FormattedMonetaryAmountProps`.